app/views/posts/show.html.erb

```diff
@@ -5,6 +5,12 @@
   <div class="right-side">
     <p> Likes: <%= Like.where('post_id = ?', @post.id).count %></p>
     <p> Comments:<%= Comment.where('post_id = ?', @post.id).count %></p>
+
+    <%if can? :destroy, @post%>
+      <button>
+        <%=link_to "Delete", @post, method: :delete%>
+      </button>
+    <%end%>
```

Comment:
- [ ] Currently, I am unable to delete the post of other users if my role is admin. Please fix this.